services/airflow/dags/update_hospitalizations/test.py

```diff
@@ -0,0 +1,14 @@
+import pandas as pd
+
+from download_hospitalizations import download_hospitalizations
+from transform_dataframe import transform_dataframe
+
+URL = "https://opendata.ecdc.europa.eu/covid19/hospitalicuadmissionrates/csv/data.csv"
+url = "http://static-files/static/hospitalizations.csv"
+Url = "../../files/static/hospitalizations.csv"
+df = download_hospitalizations(Url)
+transform = transform_dataframe(df)
+print(len(transform))
+print(transform.head(5))
+print(transform.iloc[0])
+print(transform.info())
```

Comment:
If you want to run this to get some visual feedback, I suggest wrapping it in at test and run it with 
`make watchthis TEST='test_transform_df'`. If you don't need it anymore, since the transformation is running fine, (I think it does, since your other test looks good and is passing in CI), I suggest, you throw this out.

```suggestion
def test_transform_df():
  df = download_hospitalizations(Url)
  transform = transform_dataframe(df)
  print(len(transform))
  print(transform.head(5))
  print(transform.iloc[0])
  print(transform.info())
  assert False
```